File: Iniciante/3170.cpp
#include <iostream>
using namespace std;

int main() {
    int b, g;
    cin >> b >> g;
    if(g%2 != 0) g -= 1;
    g = g/2;
    if(b >= g) cout << "Amelia tem todas bolinhas!" << endl;
    else cout << "Faltam " << (g - b) << " bolinha(s)" << endl;
    return 0;
}

File: Iniciante/C#/3170.cs
using System;
class URI {
    static void Main(string[] args) { 
        int b = int.Parse(Console.ReadLine());
        int g = int.Parse(Console.ReadLine());
        if(g%2 != 0) g -= 1;
        g = g/2;
        if(b >= g) Console.WriteLine("Amelia tem todas bolinhas!");
        else Console.WriteLine("Faltam " + (g - b) + " bolinha(s)");
    }   
}